packages/builder/builder/src/plugins/splitChunks.ts

```diff
@@ -98,6 +98,7 @@ async function splitByExperience(
       'history',
     ),
     lodash: createDependenciesRegExp('lodash', 'lodash-es'),
+    axios: createDependenciesRegExp('axios', /axios-.+/),
```

Comment:
The documentation should be updated: https://edenx.bytedance.net/builder/guide/optimization/split-chunk.html#split-by-experience

Reply:
done.